rules/windows/process_creation/proc_creation_win_priv_escalation_via_named_pipe.yml

```diff
@@ -0,0 +1,34 @@
+title: Privilege Escalation via Named Pipe Impersonation
+id: 9bd04a79-dabe-4f1f-a5ff-92430265c96b
+related:
+    - id: f35c5d71-b489-4e22-a115-f003df287317
+      type: derived
+status: experimental
+description: Detects a remote file copy attempt to a hidden network share. This may indicate lateral movement or data staging activity.
+author: Tim Rauch
+references:
+    - https://www.elastic.co/guide/en/security/current/privilege-escalation-via-named-pipe-impersonation.html
+date: 2022/09/27
+logsource:
+    category: process_creation
+    product: windows
+tags:
+    - attack.lateral_movement
+    - attack.t1021
+detection:
+    selection_name:
+        - Image|endswith:
+            - '\Cmd.Exe'
+            - '\PowerShell.EXE'
```

Comment:
Please use the standard casing for the Image name, not the OriginalFIieName one. (It doesn't matter but just for consistency)